modules/monitoring-common-terms.adoc

```diff
@@ -0,0 +1,93 @@
+The Cluster Monitoring Operator (CMO) is a central component of the monitoring stack. It deploys and manages Prometheus instances, the Thanos Querier, the Telemeter Client, and metrics targets and ensures that they are up to date. The CMO is deployed by the Cluster Version Operator (CVO).
+
+Cluster Version Operator::
+The Cluster Version Operator (CVO) manages the lifecycle of cluster Operators, many of which are installed in OpenShift Container Platform by default.
+
+config map::
+A config map provides a way to inject configuration data into pods. You can reference the data stored in a config map in a volume of type `ConfigMap`. Applications running in a pod can use this data.
+
+Container::
+It is a lightweight and executable image that comprises software and all its dependencies. Containers virtualize the operating system, as a result, you can run containers anywhere from a data center to a public or private cloud to even a developer’s laptop.
+
+custom resource (CR)::
+A CR is extension of the Kubernetes API. You can create custom resources.
+
+etcd::
+etcd is the key-value store for OpenShift Container Platform, which stores the state of all resource objects.
+
+Fluentd::
+Fluentd gathers logs from nodes and feeds them to Elasticsearch.
+
+Kubelets::
+Runs on nodes and reads the container manifests. Ensures that the defined containers have started and are running.
+
+Kubernetes API server::
+Kubernetes API server validates and configures data for the API objects.
+
+Kubernetes controller manager::
+Kubernetes controller manager governs the state of the cluster.
+
+Kubernetes scheduler::
+Kubernetes scheduler allocates pods to nodes.
+
+labels::
+You can use labels, which are key-value pairs, to organise and select subsets of objects, such as a pod.
+
+Metering::
+Metering is a general purpose data analysis tool that enables you to write reports to process data from different data sources.
+
+node::
+A worker machine in the {product-title} cluster. A node is either a virtual machine (VM) or a physical machine.
+
+Operator::
+The preferred method of packaging, deploying, and managing a Kubernetes application in an {product-title} cluster. An Operator takes human operational knowledge and encodes it into software that is packaged and shared with customers.
+
+Operator Lifecycle Manager (OLM)::
+OLM helps you to install, update, and manage the lifecycle of Kubernetes native applications. OLM is an open source toolkit designed to manage Operators in an effective, automated, and scalable way.
+
+Persistent storage::
+Stores the data even after the device is shut down. Kubernetes uses persistent volumes to store the application data
+
+Persistent volume claim (PVC)::
+You can use a PVC to mount a PersistentVolume into a Pod. You can access the storage without knowing the details of the cloud environment.
+
+pod::
+The pod is the smallest logical unit in Kubernetes. A pod contains one or more containers to run in a worker node.
```

Comment:
```suggestion
A pod is the smallest logical unit in Kubernetes. A pod contains one or more containers to run in a worker node.
```

Reply:
If you wanted to avoid the redundancy in "contains" and "containers", you could re-write the second sentence to be: A pod is comprised of one or more containers to run in a worker node.

Reply:
acked